lms/static/sass/views/_login-register.scss

```diff
@@ -356,6 +356,22 @@ $sm-btn-linkedin: #0077b5;
         text-transform: none;
         font-weight: 600;
         letter-spacing: normal;
+
+        .icon img {
+            width: auto;
+            height: auto;
+            max-height: 1.4em;
+            max-width: 1.4em;
+            margin-top: -2px;
```

Comment:
I just changed the sandbox to use the logo we need for this project, and it seems to be not vertically centered unless this margin-top is -4px. @omarkhan can you please check?

The logo as SVG is attached to OC-1374 if you want to use it for devstack testing.